### ISSUE TITLE
Make public facing Lambda Function or Endpoint/Webhook to update API?

### ISSUE THREAD

**Issue by stevelizcano** [aws-amplify/amplify-cli#2965]
>**Note**: If your question is regarding the AWS Amplify Console service, please log it in the 
[official AWS Amplify Console forum](https://forums.aws.amazon.com/forum.jspa?forumID=314&start=0)

**Which Category is your question related to?**

AWS Amplify

**Amplify CLI Version** 
4.6.0
You can use `amplify -v` to check the amplify cli version on your system

**What AWS Services are you utilizing?**
API(AppSync), Auth, Storage, Functions

**Provide additional details e.g. code snippets**

We can easily create serverless express functions with Amplify, which is awesome! 

But I'm wondering how to expose these to the internet? I have some IoT devices that collect weather data. I want to create a serverless express webhook that will take a POST request from them of the JSON data, and update API in the backend with weather data. 

How would this be possible? 

Thanks!

**Comment by kaustavghosh06** [comment]
@stevelizcano You can expose them via a REST or a GraphQL API?

**Comment by stevelizcano** [comment]
I tried created a serverless-express function using the CLI, but it didn't create a new endpoint. 

Are they deployed at the same endpoint? I.e. I do a POST request a the GraphQL endpoint and the serverless express function is automatically activated?

OR: Do I need to create a new REST API resource and use that endpoint?

**Comment by davidbiller** [comment]
You need an api. So add a api and add the correct query than with @function directive.

**Comment by kaustavghosh06** [comment]
@stevelizcano Were you able to setup an API (API Gateway or AppSync) to expose your lambda function?

**Comment by stevelizcano** [comment]
I got it to work, it wasn't working for some reason the first time. I'm going to close the issue, and probably open another one because the biggest problem (as of CLI 4.6) is that a REST-only API URL is not published by the CLI, I had to dig into the amplify backend folder to find it in the settings.

If I understand it correctly, it is only publishing (via `amplify status`) graphQL api endpoints.

Thank you for your assistantce!
